_LowLevelCode/CPP/cpp_communicator/input_parser.cpp

```diff
@@ -43,16 +43,17 @@ size_t   get_byte_length(const char*err_id, const mxArray *prhs) {
         std::stringstream buf;
         buf << " The input for " << err_id << "contains unknown vector type\n";
         mexErrMsgIdAndTxt("MPI_MEX_COMMUNICATOR:invalid_argument", buf.str().c_str());
+        // never reached but for compiler not to complain
```

Comment:
this comment isn't correct; in the default case that line will be reached.

Reply:
yes, may be should be removed.